amivapi/tests/events/test_emails.py

```diff
@@ -229,3 +229,55 @@ def test_no_nones_in_emails(self):
         mail = self.app.test_mails[1]
         for field in mail.values():
             self.assertTrue('None' not in field)
+
+    def test_moderator_reply_to(self):
+        # Check whether `reply-to` header is the moderator in email if set.
+        user = self.new_object('users')
+        user_moderator = self.new_object('users', email='xyz@gmail.com')
+        event = self.new_object('events', spots=100,
+                                selection_strategy='fcfs',
+                                moderator=user_moderator['_id'])
+
+        self.api.post('/eventsignups', data={
+            'user': str(user['_id']),
+            'event': str(event['_id'])
+        }, status_code=201).json
+
+        mail = self.app.test_mails[0]
+        self.assertTrue(mail['reply-to'] == 'xyz@gmail.com')
+
+    def test_all_emails_for_reply_to_header(self):
+        # Check that the `reply-to` header in all email is set to default
```

Comment:
same here

Reply:
This was addressed with the latest commits.